contracts/README.md

```diff
@@ -2,57 +2,54 @@
 
-### `NativeTokenSource`
-A concrete implementation of `TeleporterTokenSource` and `INativeTokenBridge` that handles the locking and release of the native EVM asset on the home chain.
+All messages sent by `TokenSpoke` instances are sent to the specified `TokenHub` contract, whether they are to redeem the collateral on the hub chain or route the tokens to another spoke chain. Routing tokens from one spoke chain to another is referred to as a "multi-hop", where the tokens are first sent back to their `TokenHub` contract to update its accounting, and then automatically routed on to their intended destination spoke.
 
-### `TeleporterTokenDestination`
-An abstract implementation of `ITeleporterTokenBridge` for a bridge contract on a destination chain that receives bridged assets from a `TeleporterTokenSource` instance. Handles receiving bridge messages from the specified token source contract to process token imports from the home chain (i.e mints), as well as burning tokens and sending messages to route them back to other chains. Once deployed, a `TeleporterTokenDestination` instance must be registered with its specified `TeleporterTokenSource` contract. This is done by calling `registerWithSource` on the destination contract, which will send a Teleporter message to the source contract with the message to register.
+`TokenSpoke` contracts allow for scaling token amounts, if the representative "wrapped" token is not a 1-to-1 equivalent of the backing asset. This token scaling can be used when the spoke asset has a higher or lower denomination than the hub asset, such as allowing for a ERC20 hub asset with a denomination of 6 to be used as the native EVM asset on a spoke chain (with a denomination of 18).
 
-All messages sent by `TeleporterTokenDestination` instances are sent to the specified token source contract, whether they are to redeem the collateral on the home or chain or route the tokens to another destination chain. Routing tokens from one destination chain to another is referred to as a "multi-hop", where the tokens are first sent back to their token source contract to update its accounting, and then automatically routed on to their intended destination.
+### `ERC20TokenSpoke`
+A concrete implementation of `TokenSpoke`, `IERC20TokenBridge`, and `IERC20` that handles the minting and burning of an ERC20 spoke chain asset. Note that the `ERC20TokenSpoke` contract _is_ the "wrapped" ERC20 implementation itself, which is why it takes the `tokenName`, `tokenSymbol`, and `tokenDecimals` in its constructor. All of the ERC20 interface implementations are inherited from the standard OpenZeppelin ERC20 implementation, and can be overriden in other implementations if desired.
```

Comment:
what are you referring to with "wrapped" here? Do you mean wrapping the hub token to the destination Subnet? Got confused with the wrapped native tokens.